### PR TITLE
ci: Add cargo vet to ensure that 3rd-party deps

### PR DESCRIPTION
**Pull request by bitboom**
This PR applies `cargo-vet` to CI

ref) https://mozilla.github.io/cargo-vet/

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -143,10 +143,17 @@ jobs:
     - name: Check pylint
       run: ./scripts/pylint.sh
 
-    - uses: actions-rs/audit-check@v1
+    - name: Audit Cargo.lock against the advisory DB
```
Comment (zpzigi754) [inline .github/workflows/ci.yml:146]: How about not using the name like the previoiusly or removing `Cargo.lock`?

Also, I think that putting this workflow at the end (e.g., after cargo-vet) would be better.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -143,10 +143,17 @@ jobs:
...
+      uses: actions-rs/audit-check@v1
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Ensure that third-party Rust dependencies
```
Comment (zpzigi754) [inline .github/workflows/ci.yml:151]: How about changing the name as `Audit third-party rust dependencies` or `Check third-party rust dependencies`?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bitboom) [reply .github/workflows/ci.yml]: Applied to the former.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bitboom) [reply .github/workflows/ci.yml]: Fixed it to the latter.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -143,10 +143,17 @@ jobs:
...
+      run: |
+        cargo install --locked cargo-vet
```
[the remark this answers is not in the document]
Comment (bitboom) [reply .github/workflows/ci.yml:148]: I referred [the installation's command](https://mozilla.github.io/cargo-vet/install.html).

**Review by zpzigi754 — APPROVED** [review]

**Pull request merged**